idseq/cli.py

```diff
@@ -87,6 +85,10 @@ def main():
         '--accept-all',
         action='store_true',
         help='Use this argument to automatically accept confirmation messages')
```

Comment:
Can you update this message to explicitly indicate that will accept all locations?
Alternatively, we could have a specific parameter `--accept-locations-match`. I actually prefer the last option.

Reply:
I will update the message. 

If we go with a separate flag, how would we describe the accept-all flag?

`Use this argument to automatically accept confirmation messages except location ones`

?

Reply:
Yeah, the all might be a bit confusing in that case.